src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,47 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const currentTabId = tabs.some(({ id }) => id === selectedTabId)
+    ? selectedTabId
+    : tabs[0].id;
+
+  const currentContent = tabs.find(({ id }) => id === currentTabId).content;
```

Comment:
```suggestion
  const currentTab = tabs.find(({ id }) => id === selectedTabId) || tabs[0];

  const currentContent = currentTab.content;
```
You don't need to check if the tab exists and only then find it, find returns undefined if not found so you can set the default value in this case.
And in this case, the currentContent is redundant